chapters/part3/fairness.md

```diff
@@ -0,0 +1,733 @@
+
+1. **Skewed sampling (feedback loop):** Future observations confirm a model's predictions, which leads to fewer opportunities to make observations that contradict the model's predictions, thereby causing any biases to compound, e.g. predictive policing tools are influenced by their own predictions of where crime will occur
+
+2. **Tainted examples:** Social biases like racism, sexism, and homophobia taint datasets, and then get baked into models which are trained and evaluated on these datasets, e.g. Amazon's automated resume screening system identified women to be unfit for engineering roles because of the company's poor track record of hiring women
+
+3. **Limited features:** Features may be less informative or less reliably collected for certain groups of people, which leads to a model exhibiting disparate errors for different groups
+
+4. **Sample size disparities:** There exist disparities in the amount of data across certain groups of people, which leads to a model exhibiting disparate errors for different groups
+
+5. **Proxies:** Even after the removal of a feature which a model should not consider (e.g. race), there exist other features with a high correlation with the undesirable feature that the model can (unfairly) leverage (e.g. zip code)
+
+Please note that every dataset is biased, and it is <em>impossible</em> to fully remove biases from datasets; even though it may be relatively easy to identify and remove explicit social biases, it is difficult to eliminate skewed samples and proxies that could reinforce these biases. That being said, it's still a great idea to create [datasheets for your datasets](https://api.semanticscholar.org/CorpusID:4421027).
+
+## Biases in Models
+
+You may often hear, in the context of biases, "It's not the model, it's the data!" This is far from the truth. Models are excellent at <em>encoding</em> and <em>amplifying</em> biases. 
+
+1. **Imbalanced datasets:** Model performance inherently favors groups with rich features and a large, diverse sample, thereby marginalizing groups with limited features and a poor sample size. 
+
+2. **Spurious correlations:** Models learn complex, non-linear and spurious correlations between features and annotations that often reflect social biases and are difficult to detect and eliminate. For example, [Wang et al.](https://api.semanticscholar.org/CorpusID:195847929) discovered that even after removing subjects from images, a model could still easily predict the gender of subjects because women were photographed overwhelmingly frequently in the kitchen, that is, the model leveraged the existence of a kitchen in the image as a proxy for gender.
+
+3. **Contextualization:** LSTMs and attention-based models, through contextualization, allow for biases to be propagated across words in a sentence.
+
+Balanced datasets are not effective at taming bias amplification because of latent variables.
```

Comment:
"Balanced datasets alone"?  I might provide a teaser for what else we need to do in a single sentence.  "We also need to..."